.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_API_URL=http://localhost:3500
+API_KEY=552737677385964
+CLOUD_NAME=dtvmeyqfx
+API_SECRET=1g_7AsLvmmaO11R_gohu-ErVks0
```

Comment:
What is this?
Looks like some secrets that shouldn't necessarily live openly on github.

Reply:
ohwoops! 

I thought I added that file to .gitignore, I'll look into this and redo the credentials

Reply:
I saw that! Looks like this file was committed before the gitignore was, no worries just need to recreate your keys and delete this file and you should be good